compiler-core/src/parse/tests.rs

```diff
@@ -631,6 +631,16 @@ pub fn main() -> Nil {
     );
 }
 
+#[test]
+fn attributes_with_no_definition() {
+    assert_module_error!(
+        r#"
+@deprecated("1")
+@target(erlang)
+"#
+    );
+}
```

Comment:
Could we have test for the original issue reported? That is having an external on a non-function definition.